### DISCUSSION TITLE
Error: cannot create std::vector larger than max_size()

### DISCUSSION THREAD

**Discussion by ppandit95** [peridigm/peridigm#282] [Q&A]
Dear @djlittl , @johntfoster , @jritta , @diehlpk 

I am trying to run the example code provided but I am getting the error mentioned in subject line which I believe is because I haven't specified explicitly the nature of variable to be used while configuring the peridigm. Any headers in this regard will be helpful.

Many Thanks
Pushkar

**Comment by jritta** [comment]
Dear Pushkar,

Unfortunately, I don't understand your comment at all. 

- What sample code are you talking about?
- What kind of error message is that? Likely a compiler message, right?

If you give us much more information we can maybe help.

Best,
Jonas

**Reply by ppandit95** [reply]
I was able to compile the peridigm without any errors but I am getting this max_size runtime error when I run the tensile test example .The output message upon running the yaml file is -

No OpenFabrics connection schemes reported that they were able to be
used on a specific port.  As such, the openib BTL (OpenFabrics
support) will be disabled for this port.

  Local host:           cmslab
  Local device:         irdma0
  Local port:           1
  CPCs attempted:       udcm
--------------------------------------------------------------------------

-- Peridigm
-- version 1.5.0 (Dev)

cannot create std::vector larger than max_size()
Wallclock Time (seconds):
  Total             0.98

Hope this helps any any header in this direction will prove beneficial.

Many Thanks
Pushkar

**Reply by jritta** [reply]
This looks to me like a problem caused by the combination of the hardware used and Peridigm (or rather MPI). Maybe you discuss this with your local hardware administrator.

Best
Jonas
